crates/core/src/network/handler/lists.rs

```diff
@@ -18,26 +19,53 @@ use lib3h_protocol::{
 };
 use std::sync::Arc;
 
-#[autotrace]
-#[holochain_tracing_macros::newrelic_autotrace(HOLOCHAIN_CORE)]
-pub fn handle_get_authoring_list(get_list_data: GetListData, context: Arc<Context>) {
+//#[autotrace]
+//#[holochain_tracing_macros::newrelic_autotrace(HOLOCHAIN_CORE)]
+pub fn handle_get_authoring_list(_get_list_data: GetListData, context: Arc<Context>) {
     let c = context.clone();
     let closure = async move || {
-        let address_map = create_authoring_map(context.clone());
-
-        let action = Action::RespondAuthoringList(EntryListData {
+        // let address_map = create_authoring_map(context.clone());
+
+        // currently sim2h asks for the authoring list and treats it just the same
+        //as the gossiping list, i.e. as data that you hold.  This is a problem because
+        //it means that gossiping isn't actually working right.  So instead of fixing that in
```

Comment:
Woah, if sim2h is doing, that is a problem and I would consider it a defect.

Last time I looked, sim2h was using the authoring list to check if this node has something new that the DHT does not store yet, such that sim2h would request that data from the node and publish it to the according authorities.

Reply:
Yes, that was incorrect. We could, in the sim2h case actually treat it that way, and we could add to sim2h the tracking of who authored what so that it does treat authors as authoritative on the DHT.   The hack of having the conductor simply republish everything it authored when it receives the authoring list request serves the exact same purpose as what say above as long as sim2h also checks to not send the data again to nodes that it knows it's already sent the publish data too.  I will be adding that.